src/Core/Context/CurrentContext.cs

```diff
@@ -360,6 +360,12 @@ public async Task<bool> ManageResetPassword(Guid orgId)
                         && (o.Permissions?.ManageResetPassword ?? false)) ?? false);
         }
 
+        public async Task<bool> ManageBilling(Guid orgId)
+        {
+            return await OrganizationOwner(orgId) &&
+                (await ProviderUserForOrgAsync(orgId) || ProviderIdForOrg(orgId) == null);
```

Comment:
Should we extract this to a helper method to make it more clear what we are checking? 

`ManagedByProvider(orgId)`

Reply:
This is the helper method, I'm not sure there's much point in adding yet more methods to `CurrentContext` to abstract this further.

This doesn't feel significantly better to me:
```
return await OrganizationOwner(orgId) &&
                (await ProviderUserForOrgAsync(orgId) || !(await ManagedByProvider(orgId));
```

and if you try to encapsulate more than that, you need to name it something like

```
return await OrganizationOwner(orgId) &&
                !(await ManagedByProviderAndImNotAProviderUser(orgId));
```